examples/tutorials/tutorial-google-driving-2.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Driving directions to clicked point | CartoDB.js</title>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+    <link rel="shortcut icon" href="http://cartodb.com/assets/favicon.ico" />
+    <style>
+      html, body, #map {
+        height: 100%;
+        padding: 0;
+        margin: 0;
+      }
+    </style>
+
+    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.12/themes/css/cartodb.css" />
+  </head>
+  <body>
+    <div id="map"></div>
+
+    <!-- include google maps library -->
+    <script type="text/javascript" src="http://www.maps.google.com/maps/api/js?sensor=false&v=3.12"></script>
+
+    <!-- include cartodb.js library -->
+    <script src="http://libs.cartocdn.com.s3.amazonaws.com/cartodb.js/v3/3.12/cartodb.js"></script>
+
+    <script>
+      var map;
+      var google_api_key = "AIzaSyCADWr4a6NraGN1ldmcBKN4W_6c6teuImw"
+      function main() {
+        // Map center
+        var myLatlng = new google.maps.LatLng(37.753, -122.433);
+        var myOptions = {
+          zoom: 13,
+          center: myLatlng, 
+          disableDefaultUI: true,
+          mapTypeId: google.maps.MapTypeId.ROADMAP
+        }
+        // Render basemap
+        map = new google.maps.Map(document.getElementById("map"), myOptions);
+        // Create services for later rendering of directions
+        var directionsDisplay = new google.maps.DirectionsRenderer();
+            directionsDisplay.setMap(map);
+        var directionsService = new google.maps.DirectionsService();
+        // The location of the Exploratorium
+        var exploratorium = new google.maps.LatLng(37.801434, -122.397561);
+        // Our CartoDB visualization
+        var vizjson_url = "http://documentation.cartodb.com/api/v2/viz/4a885510-d6fb-11e4-aedb-0e4fddd5de28/viz.json";
+        cartodb.createLayer(map, vizjson_url)
+        .addTo(map)
+        .done(function(layers) {
+            var subLayer = layers.getSubLayer(0);
+            // Change our SQL applied to the layer to include Lat and Lon values in the response
+            subLayer.set({"interactivity":"cartodb_id, lon, lat", "sql": "SELECT *, ST_X(the_geom) lon, ST_Y(the_geom) lat FROM schools_public_pt"})
```

Comment:
do you need to get lat, lon from interactivity when featureClick provides you a latlng object with the clicked position ? it's not that exact but it could work